libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -300,6 +302,11 @@ export class SkyInputBoxComponent
     this.#changeRef.markForCheck();
   }
 
+  public hideHintText(hide: boolean): void {
```

Comment:
Passing a boolean to `hideHintText` changes the meaning of the method (it can either "hide" or "show"). We could rename the method to `setHintTextHidden(hidden: boolean): void` (to match the `setHintText` method) so the meaning stays the same regardless of the boolean parameter.

Reply:
Updated to `setHintTextHidden`